code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -18,7 +18,9 @@
 	var/tracked_access = "It doesn't look like it's ever been used."
 	health = 50
 
-
+/obj/item/weapon/storage/lockbox/proc/damage(var/obj/item/I)
+//	if(istype(I, /obj/item/weapon/gun))
+//		I.damaged = 1  //why doesn't this woooork
```

Comment:
I'm probably missing something really obvious, because this is giving an undefined var error, and I don't know why